### PR TITLE
IDE-20497 ESDL.bat not handling include files

### PR DESCRIPTION
**Pull request by dehilsterlexis**
Fixed IDE-20497

Signed-off-by: dehilsterlexis <david.dehilster@lexisnexis.com>

### CODE REVIEW

**Comment by dehilsterlexis** [comment]
@rpastrana for review

**Review by rpastrana — APPROVED** [review]
@dehilsterlexis looks fine.

File: clienttools/IDEPlugins/ESDL/esdl.bat.in [hunk 1]
```diff
@@ -57,8 +57,8 @@ for /f "tokens=1,2,3 delims=/:" %%a in ("%8") do set http=%%a&set server=%%b&set
 echo.server: %server% >> %TMP%\log.txt
 echo.port  : %port% >> %TMP%\log.txt
 
-echo "%clienttoolsbindir%\esdl.exe publish %newinput% %2 --server %server% --port %port% --username %6 --password %9 -v" >>%TMP%\log.txt
-"%clienttoolsbindir%"\esdl.exe publish %newinput% %2 --server %server% --port %port% --username %6 --password %9 -v 2>%5 >>%TMP%\log.txt
+echo "%clienttoolsbindir%\esdl.exe publish %newinput% %2 --include-path %7 --server %server% --port %port% --username %6 --password %9 -v" >>%TMP%\log.txt
+"%clienttoolsbindir%"\esdl.exe publish %newinput% %2 --include-path %7 --server %server% --port %port% --username %6 --password %9 -v 2>%5 >>%TMP%\log.txt
```
Comment (richardkchapman) [inline clienttools/IDEPlugins/ESDL/esdl.bat.in]: Does this handle spaces in the include path correctly?

Comment (rpastrana) [reply clienttools/IDEPlugins/ESDL/esdl.bat.in]: Good point, %7 should be wrapped in double quotes...

Comment (dehilsterlexis) [reply clienttools/IDEPlugins/ESDL/esdl.bat.in]: @richardkchapman @rpastrana the double quotes are sent to the batch file already included. Here is the line of the IDE C++ code showing that. The folders are variable #8 in the following format statement: 

std::_tstring cmd = (boost::_tformat(_T("cmd /c %1% %2% %3% %4% %5% %6% %7% **\"%8%\"** \"%9%\" \"%10%\" \"%11%\"")) %

Comment (richardkchapman) [reply clienttools/IDEPlugins/ESDL/esdl.bat.in]: Not sure that's enough - though I'm not that familiar with how Windows cmd handles parameters. Have you tested it with a path that included spaces to be sure?

**Pull request merged**